src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -47,6 +49,40 @@ public Optional<String> getDerivedTableName(UUID entityTypeId) {
       .fetchOptional(derivedTableNameField);
   }
 
+  public List<EntityTypeColumn> fetchNamesForSingleCheckbox(UUID entityTypeId) {
+    log.info("Getting derived table name for entity type ID: {}", entityTypeId);
+    String sourceViewName = getDerivedTableName(entityTypeId).get();
+    List<EntityTypeColumn> entityColumnsList = new ArrayList<>();
+
+    String requiredFieldName = "jsonb" + "->>'name'";
+    String abc =  "jsonb" + "->>'refId'";
+    String whereClauseName = "jsonb" + "->>'type'";
```

Comment:
Tiniest of nits: all of these `"jsonb" + "->>'something'"` strings might as well be made into simple string constants (i.e., `jsonb ->> 'something'`). It doesn't really matter, since the compiler should do the concatenation at compile-time, but it just looks cleaner and will probably keep sonarqube from complaining about a duplicate string